python/cuml/test/test_preprocessing.py

```diff
@@ -24,13 +24,15 @@
     PolynomialFeatures as cuPolynomialFeatures, \
     SimpleImputer as cuSimpleImputer, \
     RobustScaler as cuRobustScaler, \
-    KBinsDiscretizer as cuKBinsDiscretizer
+    KBinsDiscretizer as cuKBinsDiscretizer 
```

Comment:
Extra space at the end; our linters should pick it up, but mentioning it since I noticed it.